spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/async/TraceDelegate.java

```diff
@@ -52,7 +52,7 @@ protected Span startSpan() {
 
 	protected String getSpanName() {
 		return this.name == null ?
-				ASYNC_COMPONENT + ":" + Thread.currentThread().getName()
```

Comment:
kindof a crappy name :P if span name ~ operation name, the name async doesn't really clarify anything especially as there's a component tag already named async. Even the thread name isn't that great. Why not fallback to toString on the delegate?